### DISCUSSION TITLE
my chia stuck in clawback

### DISCUSSION THREAD

**Discussion by carnivorouswombat** [Chia-Network/chia-blockchain#16172] [Support]
Hi,
I tried to transfer chia to kucoin for the first time a couple of days ago and used a 5hr time frame.
I have the latest version 2. I have tried deleting and re-installing all to no avail. I cannot find very much information on the web about how to fix this issue, though i see quite a few people have the same problem.

**Comment by BrandtH22** [comment]
The clawback features sends coins to a clawback address, the sender can clawback the coins at anytime using the GUI or CLI but the recipient can only claim the coins after the clawback timeout expires.

Likely kucoin does not have their system setup to autoclaim clawback transactions so you will need to clawback the transaction and send the coins to kucoin using the standard transaction.

If you are not seeing the pending clawback to claw it back then I recommend upgrading to 2.0.0 and resyncing the wallet by either deleting the wallet db or using the option in settings / advanced

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
